### PR TITLE
Tasks 938-954 Indic Glue

### PR DESCRIPTION
**Pull request by tanay2001**

### CODE REVIEW

**Comment by danyaljj** [comment]
Thanks for the PR! 

@Sujan242  wondering if you can review this PR?

**Comment by Sujan242** [comment]
Hey,

Can you change the task numbers to 938-954 as task 937 is a part of PR #352?
Also, there seems to be a merge conflict. Please resolve them too.

**Comment by danyaljj** [comment]
@swarooprm I don't speak any of the languages here. Do we have any native speakers in any of these languages that can provide feedback for this PR?

**Comment by swarooprm** [comment]
> @swarooprm I don't speak any of the languages here. Do we have any native speakers in any of these languages that can provide feedback for this PR?

I know some of the languages, I am reviewing them. Will pass it on to others for the languages I do not know.

**Comment by aarunku5** [comment]
In the readme, can you put your tasks numerical order?

938- In the explanations, say why exactly the choice is the better answer. In the definition: "alternative that is a"--->"alternative that is either a".  Also, specify that the premise is कथन: field and the two alternatives are the \"विकल्पA\" and \"विकल्प B\" fields.
939- In the explanations, say why the second sentence is a cause or effect. In the definition: "your"-->you're. 
940, 942-  Same comments as for 938.
941, 943- Same comments as for 939.
944- Capitalize "Assamese". Specify that your output format should be the correct word from the 4 options. In all your explanations, you need to specify why the option is correct.
945-954- Same comments as for 944.
948- The language is Odia, not Odic.

**Comment by tanay2001** [comment]
> In the readme, can you put your tasks numerical order?
> 
> 938- In the explanations, say why exactly the choice is the better answer. In the definition: "alternative that is a"--->"alternative that is either a". Also, specify that the premise is कथन: field and the two alternatives are the "विकल्पA" and "विकल्प B" fields. 939- In the explanations, say why the second sentence is a cause or effect. In the definition: "your"-->you're. 940, 942- Same comments as for 938. 941, 943- Same comments as for 939. 944- Capitalize "Assamese". Specify that your output format should be the correct word from the 4 options. In all your explanations, you need to specify why the option is correct. 945-954- Same comments as for 944. 948- The language is Odia, not Odic.

Hi @aarunku5 

Thanks for the reviews, I have addressed most of them. I was confused regarding the wiki tasks, what explanation can I give when the answers are completely factual, places where logic or commonsense is involved I am adding the suitable explanation

**Comment by aarunku5** [comment]
Thanks, a few more corrections:

938:
Capitalize Hindi. 
P2: "is a bad"-->"is bad"

939:
Capitalize Hindi. 
P1: "loose money"-->"lose money"

940:
Capitalize Gujarati
P1: close-->closed
P2:change the ending--> will result in it being turned off
N2:breaking-->running


941:
Capitalize Gujarati
P1: would effect-->will cause
N1:(effect) -- delete this

942:
Capitalize Marathi

943:
The changes don't seem to have been addressed? Change similar to 939/941.

944-954:
You need to capitalize the language names 
Specify that the output format is the correct word itself from the options (to clarify it is not ABCD output)

Here's an example of how to modify the explanations:
{
                   "input": 
                           "Statement: திடீரென இது ஓடும் பாதையில் நிலத்தின் சரிவு குறைவடைவதால், வழமைக்கு மாறான ஒரு அம்சமாக 
                           உள்நாட்டுக் கழிமுகம் ஒன்றும் உருவாகியுள்ளது.  இதனால் இவ்விடத்தில் வலைப் பின்னலாக அமைந்துள்ள 
                           <MASK>களும், சதுப்பு நிலங்களும், ஏரிகளும் பெல்ஜியம் நாட்டின் பரப்பளவுக்குச் சமனான நிலப்பகுதியில் பரந்து 
                          அமைந்துள்ளன.  பருவகால ஆற்றுப் பெருக்கு வேளாண்மை, மீன்பிடித் தொழில் ஆகியவை தொடர்பில் மிகுந்த 
                          உற்பத்தித்திறன் கொண்டதாக அமைகின்றது.\n\n
                          Option A: ஆறு\n\n Option B: வடிநிலம்\n\n Option C: சிற்றாறு\n\n Option D: ஏரி",
                 "output": "சிற்றாறு",
                "explanation": "The most suitable word from the given options to replace the <MASK> token is சிற்றாறு"
}

So in this example, your explanation should be something like:
The passage talks about different water sources present in Belgium near a domestic estuary, and how they are used in agriculture and fishing. In the options given, A,B, and D refer to water bodies already referenced in the passage, while option C is rivulets, which are not mentioned in the passage. Hence, option C is the best answer.

(ofc this might not be true for all passages- i.e., there might be a more indirect reason related to the context of the passage that dictates which is the correct answer)

So basically, summarily translate the passage in a line or so, and then say why your selected option fits in better than the others.

**Comment by tanay2001** [comment]
> Thanks, a few more corrections:
> 
> 938: Capitalize Hindi. P2: "is a bad"-->"is bad"
> 
> 939: Capitalize Hindi. P1: "loose money"-->"lose money"
> 
> 940: Capitalize Gujarati P1: close-->closed P2:change the ending--> will result in it being turned off N2:breaking-->running
> 
> 941: Capitalize Gujarati P1: would effect-->will cause N1:(effect) -- delete this
> 
> 942: Capitalize Marathi
> 
> 943: The changes don't seem to have been addressed? Change similar to 939/941.
> 
> 944-954: You need to capitalize the language names Specify that the output format is the correct word itself from the options (to clarify it is not ABCD output)
> 
> Here's an example of how to modify the explanations: { "input": "Statement: திடீரென இது ஓடும் பாதையில் நிலத்தின் சரிவு குறைவடைவதால், வழமைக்கு மாறான ஒரு அம்சமாக உள்நாட்டுக் கழிமுகம் ஒன்றும் உருவாகியுள்ளது. இதனால் இவ்விடத்தில் வலைப் பின்னலாக அமைந்துள்ள களும், சதுப்பு நிலங்களும், ஏரிகளும் பெல்ஜியம் நாட்டின் பரப்பளவுக்குச் சமனான நிலப்பகுதியில் பரந்து அமைந்துள்ளன. பருவகால ஆற்றுப் பெருக்கு வேளாண்மை, மீன்பிடித் தொழில் ஆகியவை தொடர்பில் மிகுந்த உற்பத்தித்திறன் கொண்டதாக அமைகின்றது.\n\n Option A: ஆறு\n\n Option B: வடிநிலம்\n\n Option C: சிற்றாறு\n\n Option D: ஏரி", "output": "சிற்றாறு", "explanation": "The most suitable word from the given options to replace the token is சிற்றாறு" }
> 
> So in this example, your explanation should be something like: The passage talks about different water sources present in Belgium near a domestic estuary, and how they are used in agriculture and fishing. In the options given, A,B, and D refer to water bodies already referenced in the passage, while option C is rivulets, which are not mentioned in the passage. Hence, option C is the best answer.
> 
> (ofc this might not be true for all passages- i.e., there might be a more indirect reason related to the context of the passage that dictates which is the correct answer)
> 
> So basically, summarily translate the passage in a line or so, and then say why your selected option fits in better than the others.

Thanks again for the pointers, I have fixed the grammatical issues and also have added the explanations for the wiki tasks, hope its good to go now

**Comment by aarunku5** [comment]
Thanks, looks good to me @swarooprm @danyaljj
